cmd/nodecmd/create.go

```diff
@@ -291,6 +304,7 @@ func createNodes(cmd *cobra.Command, args []string) error {
 		networkoptions.NonLocalSupportedNetworkOptions,
 		"",
 	)
+	setGlobalNetworkFlags(network)
```

Comment:
I don't believe this should be needed. Remaining code should use the network object, not the flags.
Can you fix that